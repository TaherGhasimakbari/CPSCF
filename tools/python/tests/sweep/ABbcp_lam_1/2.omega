 format  1  0
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.5152163742E+00
group_name          
                '-1'
N_monomer           
                   2
N_star              
                  17
  7.800000002367E+00  7.200000008633E+00       0     1
 -5.248463895703E+00  4.883550814194E+00       1     2
  9.046910331419E-01  1.280637974575E+00       2     2
  6.145423978854E-01 -2.985370218591E-01       3     2
 -3.558135645180E-01 -4.444097862295E-01       4     2
 -1.197957286212E-01 -4.370452945981E-03       5     2
  6.787434388700E-02  8.541670709734E-02       6     2
  2.361189496479E-02  8.963819429451E-03       7     2
 -1.146357146094E-02 -1.455067569829E-02       8     2
 -4.465117639886E-03 -2.639028985046E-03       9     2
  1.731786799742E-03  2.239722728520E-03      10     2
  7.933589999200E-04  5.706196590348E-04      11     2
 -2.353977505633E-04 -3.157636113083E-04      12     2
 -1.309200345337E-04 -1.043991091642E-04      13     2
  2.860032610271E-05  4.132926811840E-05      14     2
  1.765449625960E-05  1.492728773059E-05      15     2
 -4.217287183272E-06 -6.907999788459E-06      16     1
