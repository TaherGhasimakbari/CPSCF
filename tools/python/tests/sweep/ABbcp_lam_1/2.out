 format  1  0

MONOMERS            
N_monomer           
                   2
kuhn                
    1.0000000000E+00    1.0000000000E+00

CHAINS              
N_chain             
                   1
N_block             
                   2
block_monomer       
                   1                   2
block_length        
    4.8000000022E-01    5.1999999978E-01

COMPOSITION         
ensemble            
                   0
phi_chain           
    1.0000000000E+00

INTERACTION         
interaction_type    
               'chi'
chi                 
    1.5000000011E+01

UNIT_CELL           
dim                 
                   1
crystal_system      
          'lamellar'
N_cell_param        
                   1
cell_param          
    1.5152163742E+00

DISCRETIZATION      
ngrid               
                  32
chain_step          
    1.0000000000E-02

FILE_PREFIXES       
input_prefix        
               'in.'
output_prefix       
              'ref/'

BASIS               
group_name          
                '-1'

ITERATE             
max_itr             
                  20
error_max           
    1.0000000000E-08
domain              
                   T
itr_algo            
                'NR'
N_cut               
                  17

SWEEP               
s_max               
    2.0000000000E+01
d_block_length      
   -1.0000000000E-02    1.0000000000E-02
end_increments      

FINISH              

THERMO              
f_Helmholtz         
    2.4278912978E+00
f_homo              
    2.7440000029E+00
pressure            
    3.0164253378E+00
mu_chain            
    5.4443166356E+00
stress              
    2.6529258315E-12

STATISTICS          
N_star              
                  17
Final Error         
    2.5257783920E-09
Iterations          
                   4
Basis Time          
    2.6167106628E-01
SCF Time            
    3.4641981125E-02
